### ISSUE TITLE
 TypeError: Cannot read properties of null (reading '0')

### ISSUE THREAD

**Issue by Shahdoh** [ThroughlyDruxy/tor2e-npc-parser#1]
Getting the following error when clicking on "Go" with adversary text pasted in place: 
`Uncaught Error: TypeError: Cannot read properties of null (reading '0')
[No packages detected]
    at Dialog.submit (foundry.js:43929)
    at Dialog._onClickButton (foundry.js:43888)
    at HTMLButtonElement.dispatch (jquery.min.js:2)
    at HTMLButtonElement.v.handle (jquery.min.js:2)
submit @ foundry.js:43929
_onClickButton @ foundry.js:43888
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2`

Pasted Text: 
`WILD WOLF
Keen-eyed, Grim
ATTRIBUTE LEVEL
3
ENDURANCE
12
MIGHT
1
HATE
3
PARRY
—
ARMOUR
1
COMBAT PROFICIENCIES: Fangs 3 (3/14, Pierce)
FELL ABILITIES: Fear of Fire. The creature loses 1 Hate at the
start of each round it is engaged in close combat with an
adversary wielding a torch or other sort of burning item.
Snake-like Speed. When targeted by an attack,
spend 1 Hate to make the attack roll Ill-favoured`

General Info: 
Foundry VTT Version 9.249 
The One Ring 2nd Edition 0.2.0
39 active modules

**Comment by Shahdoh** [comment]
Per Discord conversation, I did not include all of the appropriate text.  Included text needs to be from the name of the creature before the descriptive text (the part I missed) to the end of the "Fell Abilities" section. 

We also found out that the creatures name before the "descriptive" text and the name before the "attributes" need to match.

Also, information from unofficial sources may not be in the same order so some manual manipulation is required to match the official format. Pay attention to case and seperator types when doing so.

Thanks again for a greate tool!
